File: trunk/csv2code/src/shells/generate/lua/factions_bigwigs_data.lua
--
-- 派系权贵表.csv
-- 权贵编号 = id
-- 名称 = name
-- 立绘资源 = picture
-- 头像资源 = head
-- 己方派系BUFF = factionsbuff
-- 己方阵营BUFF = campbuff
-- 敌对权贵 = hostile
-- 建筑解锁等级 = limit
-- 初始势力 = initial
-- 初始势力的友好度 = friend

local data = {
    [1] = {["id"] = 1, ["name"] = "白凤", ["picture"] = 1000, ["head"] = "1000", ["factionsbuff"] = {2001}, ["campbuff"] = {2017}, ["hostile"] = {5}, ["limit"] = 1, ["initial"] = 1, ["friend"] = 20000},
    [2] = {["id"] = 2, ["name"] = "平山丘", ["picture"] = 1001, ["head"] = "1001", ["factionsbuff"] = {2002}, ["campbuff"] = {2018}, ["hostile"] = {6}, ["limit"] = 1, ["initial"] = 1, ["friend"] = 20000},
    [3] = {["id"] = 3, ["name"] = "杏林生", ["picture"] = 40014, ["head"] = "40014", ["factionsbuff"] = {2003}, ["campbuff"] = {2019}, ["hostile"] = {7}, ["limit"] = 1, ["initial"] = 2, ["friend"] = 20000},
    [4] = {["id"] = 4, ["name"] = "青松道人", ["picture"] = 40004, ["head"] = "40004", ["factionsbuff"] = {2004}, ["campbuff"] = {2020}, ["hostile"] = {8}, ["limit"] = 1, ["initial"] = 2, ["friend"] = 20000},
    [5] = {["id"] = 5, ["name"] = "白芷散人", ["picture"] = 40008, ["head"] = "40008", ["factionsbuff"] = {2005}, ["campbuff"] = {2021}, ["hostile"] = {1}, ["limit"] = 2, ["initial"] = 3, ["friend"] = 20000},
    [6] = {["id"] = 6, ["name"] = "长风君", ["picture"] = 40001, ["head"] = "40001", ["factionsbuff"] = {2006}, ["campbuff"] = {2022}, ["hostile"] = {2}, ["limit"] = 2, ["initial"] = 3, ["friend"] = 20000},
    [7] = {["id"] = 7, ["name"] = "惊玄君", ["picture"] = 40004, ["head"] = "40004", ["factionsbuff"] = {2007}, ["campbuff"] = {2023}, ["hostile"] = {3}, ["limit"] = 3, ["initial"] = 4, ["friend"] = 20000},
    [8] = {["id"] = 8, ["name"] = "松柏老人", ["picture"] = 40010, ["head"] = "40010", ["factionsbuff"] = {2008}, ["campbuff"] = {2024}, ["hostile"] = {4}, ["limit"] = 3, ["initial"] = 4, ["friend"] = 20000},
    [9] = {["id"] = 9, ["name"] = "青竹客", ["picture"] = 1000, ["head"] = "1000", ["factionsbuff"] = {2009}, ["campbuff"] = {2025}, ["hostile"] = {13}, ["limit"] = 3, ["initial"] = 5, ["friend"] = 20000},
    [10] = {["id"] = 10, ["name"] = "凌波仙子", ["picture"] = 40017, ["head"] = "40017", ["factionsbuff"] = {2010}, ["campbuff"] = {2026}, ["hostile"] = {14}, ["limit"] = 4, ["initial"] = 5, ["friend"] = 20000},
    [11] = {["id"] = 11, ["name"] = "蓬莱山主", ["picture"] = 40023, ["head"] = "40023", ["factionsbuff"] = {2011}, ["campbuff"] = {2027}, ["hostile"] = {15}, ["limit"] = 4, ["initial"] = 6, ["friend"] = 20000},
    [12] = {["id"] = 12, ["name"] = "血罗刹", ["picture"] = 40004, ["head"] = "40004", ["factionsbuff"] = {2012}, ["campbuff"] = {2028}, ["hostile"] = {16}, ["limit"] = 4, ["initial"] = 6, ["friend"] = 20000},
    [13] = {["id"] = 13, ["name"] = "璎珞女", ["picture"] = 40008, ["head"] = "40008", ["factionsbuff"] = {2013}, ["campbuff"] = {2029}, ["hostile"] = {9}, ["limit"] = 5, ["initial"] = 7, ["friend"] = 20000},
    [14] = {["id"] = 14, ["name"] = "萍水剑客", ["picture"] = 40009, ["head"] = "40009", ["factionsbuff"] = {2014}, ["campbuff"] = {2030}, ["hostile"] = {10}, ["limit"] = 5, ["initial"] = 7, ["friend"] = 20000},
    [15] = {["id"] = 15, ["name"] = "丘道人", ["picture"] = 40009, ["head"] = "40009", ["factionsbuff"] = {2015}, ["campbuff"] = {2031}, ["hostile"] = {11}, ["limit"] = 6, ["initial"] = 8, ["friend"] = 20000},
    [16] = {["id"] = 16, ["name"] = "落雁客", ["picture"] = 40016, ["head"] = "40016", ["factionsbuff"] = {2016}, ["campbuff"] = {2032}, ["hostile"] = {12}, ["limit"] = 6, ["initial"] = 8, ["friend"] = 20000}
}

return data

File: trunk/csv2code/src/shells/generate/java/FactionsBigwigsData.java
package com.gamejelly.gong2.config.data;

import com.gamejelly.gong2.config.data.base.LData;
import com.gamejelly.gong2.config.data.base.LMap;

/**
 * 派系权贵表.csv
 * 权贵编号 = id
 * 名称 = name
 * 己方派系BUFF = factionsbuff
 * 己方阵营BUFF = campbuff
 * 敌对权贵 = hostile
 * 建筑解锁等级 = limit
 * 初始势力 = initial
 * 初始势力的友好度 = friend
 */
public class FactionsBigwigsData extends LData {
    public static final LMap data = new LMap();

    static {
        init0();
        init1();
        init2();
        init3();
        data.setFrozen(true);
    }

    private static void init0() {
data.map(
    1, map("id", 1, "name", "白凤", "factionsbuff", list(2001), "campbuff", list(2017), "hostile", list(5), "limit", 1, "initial", 1, "friend", 20000),
    2, map("id", 2, "name", "平山丘", "factionsbuff", list(2002), "campbuff", list(2018), "hostile", list(6), "limit", 1, "initial", 1, "friend", 20000),
    3, map("id", 3, "name", "杏林生", "factionsbuff", list(2003), "campbuff", list(2019), "hostile", list(7), "limit", 1, "initial", 2, "friend", 20000),
    4, map("id", 4, "name", "青松道人", "factionsbuff", list(2004), "campbuff", list(2020), "hostile", list(8), "limit", 1, "initial", 2, "friend", 20000),
    5, map("id", 5, "name", "白芷散人", "factionsbuff", list(2005), "campbuff", list(2021), "hostile", list(1), "limit", 2, "initial", 3, "friend", 20000)
);
    }

    private static void init1() {
data.map(
    6, map("id", 6, "name", "长风君", "factionsbuff", list(2006), "campbuff", list(2022), "hostile", list(2), "limit", 2, "initial", 3, "friend", 20000),
    7, map("id", 7, "name", "惊玄君", "factionsbuff", list(2007), "campbuff", list(2023), "hostile", list(3), "limit", 3, "initial", 4, "friend", 20000),
    8, map("id", 8, "name", "松柏老人", "factionsbuff", list(2008), "campbuff", list(2024), "hostile", list(4), "limit", 3, "initial", 4, "friend", 20000),
    9, map("id", 9, "name", "青竹客", "factionsbuff", list(2009), "campbuff", list(2025), "hostile", list(13), "limit", 3, "initial", 5, "friend", 20000),
    10, map("id", 10, "name", "凌波仙子", "factionsbuff", list(2010), "campbuff", list(2026), "hostile", list(14), "limit", 4, "initial", 5, "friend", 20000)
);
    }

    private static void init2() {
data.map(
    11, map("id", 11, "name", "蓬莱山主", "factionsbuff", list(2011), "campbuff", list(2027), "hostile", list(15), "limit", 4, "initial", 6, "friend", 20000),
    12, map("id", 12, "name", "血罗刹", "factionsbuff", list(2012), "campbuff", list(2028), "hostile", list(16), "limit", 4, "initial", 6, "friend", 20000),
    13, map("id", 13, "name", "璎珞女", "factionsbuff", list(2013), "campbuff", list(2029), "hostile", list(9), "limit", 5, "initial", 7, "friend", 20000),
    14, map("id", 14, "name", "萍水剑客", "factionsbuff", list(2014), "campbuff", list(2030), "hostile", list(10), "limit", 5, "initial", 7, "friend", 20000),
    15, map("id", 15, "name", "丘道人", "factionsbuff", list(2015), "campbuff", list(2031), "hostile", list(11), "limit", 6, "initial", 8, "friend", 20000)
);
    }

    private static void init3() {
data.map(
    16, map("id", 16, "name", "落雁客", "factionsbuff", list(2016), "campbuff", list(2032), "hostile", list(12), "limit", 6, "initial", 8, "friend", 20000)
);
    }

}